src/ucp/core/ucp_request.inl

```diff
@@ -86,6 +89,23 @@
     }
 
 
+#define ucp_request_pre_init(_req) (_req)->local_req_id = UCP_REQUEST_ID_INVALID
```

Comment:
inline func
also need a better name for it.. maybe ucp_request_reset

Reply:
removed